ansible/files/admin_api_scripts/pg_upgrade_scripts/complete.sh

```diff
@@ -99,6 +99,11 @@ function complete_pg_upgrade {
         CI_start_postgres --new-bin
     fi
 
+    # https://supabase.slack.com/archives/C02BJ2239GA/p1721462356684549
```

Comment:
Remove internal link?

Reply:
to follow other upgrade files pattern, maybe like this:

```bash
if [ -f "/var/lib/postgresql/extension/wrappers--0.3.1--0.4.1.sql" ]; then
    cp /var/lib/postgresql/extension/wrappers--0.3.1--0.4.1.sql /var/lib/postgresql/extension/wrappers--0.3.0--0.4.1.sql
    ln -s /var/lib/postgresql/extension/wrappers--0.3.0--0.4.1.sql /usr/share/postgresql/15/extension/wrappers--0.3.0--0.4.1.sql
fi
```

Reply:
Thanks both! Refactored this a bit